astropy/modeling/tests/test_constraints.py

```diff
@@ -208,7 +209,7 @@ def test_bounds_gauss2d_lsq(self, fitter):
             ctx2 = nullcontext()
             if isinstance(fitter, fitting.TRFLSQFitter):
                 ctx = np.errstate(invalid="ignore", divide="ignore")
-                if not NUMPY_LT_2_0:
+                if not NUMPY_LT_2_0 or minversion("scipy", "1.11.2"):
```

Comment:
Please do something like this for consistency. 
```suggestion
                if not NUMPY_LT_2_0 or not SCIPY_LT_1_11_2:
```

The wcsaxes failure is unrelated and we can ignore it in this PR. Thanks!